scraper/.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Tests
```

Comment:
Heb deze workflows alvast toegevoegd, maar zo te zien werken ze helaas alleen als ze in de repo root staan. Voor nu laat ik ze gewoon hier staan, als we dan deze folder naar een eigen repo splitsen zal dit vanzelf werken.